jsapp/js/account/useUsage.hook.ts

```diff
@@ -46,11 +49,17 @@ export function useUsage() {
     if (!usage.isPeriodLoaded) {
       return;
     }
-
     getUsageForOrganization().then((data) => {
       if (!data) {
         return;
       }
+      let lastUpdated: UsageState['lastUpdated'] = null;
+      if ('headers' in data && data.headers instanceof Headers) {
```

Comment:
Could be it simplified with `if (data?.headers instanceof Headers)`?

Reply:
Unfortunately, TS complains here without `in` acting as a type guard:

![image](https://github.com/kobotoolbox/kpi/assets/7819986/1dc2d185-65f0-46ca-8666-3b883dc54b27)

![image](https://github.com/kobotoolbox/kpi/assets/7819986/aae73ee7-5a28-41c0-8d8f-9dd5c705b6aa)